derive_builder/src/options/struct_mode.rs

```diff
@@ -233,7 +230,7 @@ impl From<OptionsBuilder<StructMode>> for (StructOptions, OptionsBuilder<FieldMo
                 `field(<vis>)` to the builder attribute on the struct. (Found {})",
                 where_diagnostics));
         }
-        
+
```

Comment:
My editor keeps insisting on doing this, and I'm not sure why. I'll see if I can find a setting that will make it stop.